pkg/terminal/command.go

```diff
@@ -368,6 +369,19 @@ Defines <alias> as an alias to <command> or removes an alias.`},
 	
 If locspec is omitted edit will open the current source file in the editor, otherwise it will open the specified location.`},
 		{aliases: []string{"libraries"}, cmdFn: libraries, helpMsg: `List loaded dynamic libraries`},
+
+		{aliases: []string{"examinemem", "x"}, cmdFn: examineMemoryCmd, helpMsg: `Examine memory: x /[<FMT>] ADDRESS.
```

Comment:
1. This description makes it look like the '/' should always be included, i.e. that to use the default for FMT you should type `x / ADDRESS`. 
2. It should use the full name of the command (`examinemem`) instead of `x` for the syntax.
3. To be consistent with the way we describe syntax elsewhere both FMT and ADDRESS should be lowercase and inside angular parenthesis.
4. We don't use a `/` for options anywhere else, it should be a `-` (I get it that it's following gdb syntax but I would prefer internal consistency to external consistency).

Reply:
Done